### PR TITLE
Devnet ITHEUM-a61317 new token branding + ITHEUM-df6f26 Token Branding Updates

### PR DESCRIPTION
**Pull request by newbreedofgeek**
- ITHEUM devnet token branding (https://devnet-explorer.elrond.com/tokens/ITHEUM-a61317)
- ITHEUM mainnet token image update and CMC and CoinGecko links

### CODE REVIEW

File: tokens/ITHEUM-df6f26/info.json [hunk 1]
```diff
@@ -21,4 +23,4 @@
     "ITHWEGLDF-8fd842",
     "SITHEUM-e05083"
   ]
-}
+}
```
Comment (bogdan-rosianu) [inline tokens/ITHEUM-df6f26/info.json:26]: optional: empty line at the end of the file

File: devnet/tokens/ITHEUM-a61317/info.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "blog": "https://itheum.medium.com",
+    "twitter": "https://twitter.com/itheum",
+    "telegram": "https://t.me/itheum",
+    "discord": "https://itheum.io/discord",
+    "whitepaper": "https://itheum.io/whitepaper",
+    "coinmarketcap": "https://coinmarketcap.com/currencies/itheum",
+    "coingecko": "https://www.coingecko.com/en/coins/itheum"
+  },
+  "status": "active"
+}
```
Comment (bogdan-rosianu) [inline devnet/tokens/ITHEUM-a61317/info.json:14]: optional: empty line at the end of the file

File: tokens/ITHEUM-df6f26/info.json (hunk 1 above)
Comment (newbreedofgeek) [reply tokens/ITHEUM-df6f26/info.json]: Fixed

File: devnet/tokens/ITHEUM-a61317/info.json (hunk 2 above)
Comment (newbreedofgeek) [reply devnet/tokens/ITHEUM-a61317/info.json]: Fixed

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**